tests/test1.nim

```diff
@@ -0,0 +1,70 @@
+import unittest
+import strutils
+
+import html_dsl
+
+func htmlhash(page: string): string =
+  ## Removes whitespace, for equality checking
+  ## Dumb? Genius? Someone cleverer than me could use `xmltree`
+  page.replace(" ", "").replace("\n", "")
```

Comment:
I tried many different ways, but I'm new to Nim, and couldn't easily see how to get this to work. In hindsight, maybe `parsehtml` would work?